utils/colors.ts

```diff
@@ -1,30 +1,32 @@
+import { Contract } from "@ethersproject/contracts";
+
 const chalk = require('chalk');
 
-function dim() {
+export function dim(msg: any) {
```

Comment:
`msg` should be a `string`, I don't think we pass any other type to it. Same for the other functions below.
```suggestion
export function dim(msg: string) {
```